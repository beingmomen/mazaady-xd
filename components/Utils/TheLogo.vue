<template>
  <div class="flex flex-row-reverse items-center gap-1">
    <img src="@/assets/images/logo/icon.svg" alt="icon" loading="lazy" />
    <div class="flex flex-col items-end mt-2">
      <span class="flex flex-row-reverse">
        <img src="@/assets/images/logo/m.svg" alt="icon" loading="lazy" />
        <img src="@/assets/images/logo/a.svg" alt="icon" loading="lazy" />
        <img src="@/assets/images/logo/z.svg" alt="icon" loading="lazy" />
        <img src="@/assets/images/logo/a2.svg" alt="icon" loading="lazy" />
        <img src="@/assets/images/logo/a3.svg" alt="icon" loading="lazy" />
        <img src="@/assets/images/logo/d.svg" alt="icon" loading="lazy" />
        <img src="@/assets/images/logo/y.svg" alt="icon" loading="lazy" />
      </span>
      <img src="@/assets/images/logo/back.svg" class="mt-1" alt="icon" loading="lazy" />
    </div>
  </div>
</template>

<script>
export default {}
</script>

<style lang="scss" scoped>
</style>
