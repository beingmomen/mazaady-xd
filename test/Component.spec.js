import { mount } from '@vue/test-utils'
import TheLogo from '@/components/Utils/TheLogo.vue'

describe('TheLogo', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(TheLogo);
    expect(wrapper.vm).toBeTruthy();
  });

  test('renders correctly', () => {
    const wrapper = mount(TheLogo);
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('renders all images', () => {
    const wrapper = mount(TheLogo);
    const images = wrapper.findAll('img');
    expect(images).toHaveLength(9);
  });
});

