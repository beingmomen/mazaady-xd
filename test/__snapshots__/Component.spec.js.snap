// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheLogo renders correctly 1`] = `
"<div class="flex flex-row-reverse items-center gap-1"><img src="@/assets/images/logo/icon.svg" alt="icon" loading="lazy">
  <div class="flex flex-col items-end mt-2"><span class="flex flex-row-reverse"><img src="@/assets/images/logo/m.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/a.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/z.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/a2.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/a3.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/d.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/y.svg" alt="icon" loading="lazy"></span> <img src="@/assets/images/logo/back.svg" alt="icon" loading="lazy" class="mt-1"></div>
</div>"
`;

exports[`renders correctly 1`] = `
"<div class="flex flex-row-reverse items-center gap-1"><img src="@/assets/images/logo/icon.svg" alt="icon" loading="lazy">
  <div class="flex flex-col items-end mt-2"><span class="flex flex-row-reverse"><img src="@/assets/images/logo/m.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/a.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/z.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/a2.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/a3.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/d.svg" alt="icon" loading="lazy"> <img src="@/assets/images/logo/y.svg" alt="icon" loading="lazy"></span> <img src="@/assets/images/logo/back.svg" alt="icon" loading="lazy" class="mt-1"></div>
</div>"
`;
